src/utils/wasm.js

```diff
@@ -8,6 +8,7 @@ type WasmState = {
   offsets: Array<number>
 };
 
+// $FlowIgnore
```

Comment:
I use `FlowIgnore` because the flow errors produced on line 12 seem to be related to a bug from https://github.com/facebook/flow/issues/4742.

I tried replacing `WasmState ` with `any`, but it still gave me errors.
<img width="755" alt="screen shot 2017-11-05 at 11 13 04 am" src="https://user-images.githubusercontent.com/4562118/32411716-8b997fa6-c21d-11e7-99ee-6764f1ea7da0.png">